CODEOWNERS

```diff
@@ -1,88 +1,255 @@
+# Instructions for CODEOWNERS file format and automatic build failure notifications:
+# https://github.com/Azure/azure-sdk/blob/main/docs/policies/opensource.md#codeowners
+
+#########
+# Specification
+#########
+
+# PRLabel: %Cognitive Services
 /dev/cognitiveservices/data-plane/Language/ @assafi @rokulka @ChongTang @annatisch @heaths @Azure/api-stewardship-board
+
 /specification/adp/ @Azure/adp
+
+# PRLabel: %Analysis Services
 /specification/analysisservices/ @taiwu
+
+# PRLabel: %Alerts Management
 /specification/alertsmanagement/ @ofirmanor @olalavi @erangon @orieldar @ilaizi @shakednai1 @khaboasb @orenhor
+
+# PRLabel: %API Management
 /specification/apimanagement/ @promoisha @solankisamir
```

Comment:
@ruowan @akning-ms unrelated to this PR but there are a lot of invalid users in this code owners file that we should figure out how to get fixed. Can you guys please file a clean-up item for tracking this issue.

Reply:
We create a issue to track cleanup code owner files. https://github.com/Azure/azure-sdk-tools/issues/3374